src/meta-srv/src/procedure/wal_prune.rs

```diff
@@ -119,6 +120,7 @@ impl WalPruneProcedure {
         if !check_heartbeat_collected_region_ids(&region_ids, &flush_entry_ids_map)
             || region_ids.is_empty()
         {
+            warn!("The heartbeat collected region ids do not contain all region ids in the topic-region map. Aborting the WAL prune procedure.");
```

Comment:
Let's introducing the topic name, actual region number  and expected region number

Reply:
+1